src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -364,16 +369,55 @@ HttpURLConnection getStacksHttpURLConn(URL stacksUrl, String requestMethod)
         return stacksConn;
     }
 
-    static String urlEncode(String urlComponent)
+    /** We define these escape methods, because the most obvious method to use (URLEncoder.encode) works fine
```

Comment:
fair enough.  i just figured: encoding issues are squirrelly and can be hard to diagnose, and it took a few hours of sorting through vague and contradictory documentation and discussion, and some trial and error to get things working.  it wouldn't be unreasonable for a future developer to change the way encoding's done there, or to ask why the built in URLEncoder isn't being used, so it felt like it'd be nice to save our future selves some possible headaches/puzzlement/research.